package collision_message_plus.msgs;
// import "gz_string.proto";
import "pose.proto";

message CollisionMessage
{
	required string collision1 = 1;
	required string collision2 = 2;
	required gazebo.msgs.Pose PositionCol1 = 3;
	optional gazebo.msgs.Pose PositionCol2 = 4;
}